### ISSUE TITLE
v4.1 color regressions

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2584]
- [x] [syntax error](https://github.com/ManifoldScholar/manifold/blob/master/client/src/theme/styles/base/_appearance.scss#L33) in CSS variable (needs comma)
- [x] `rgba()` function doesn't work with hexcode format (to reproduce, try moving keyboard focus to BE "Save Project" button)
<img width="468" alt="Screen Shot 2020-01-22 at 2 46 13 PM" src="https://user-images.githubusercontent.com/12454203/72941536-0dee6400-3d26-11ea-8657-1dac8835c236.png">

- [x] `.button-primary` needs different bg color on frontend
<img width="195" alt="Screen Shot 2020-01-22 at 2 51 23 PM" src="https://user-images.githubusercontent.com/12454203/72941774-adabf200-3d26-11ea-9bbc-39d7af88225c.png">

- [x] "New" icon has hard-coded `fill` value of Manifold green
- [x] hex code for Manifold green is also hard-coded in the `PressHeader` and `ResourcePlayerAudio` components
- [x] FE search filter focus state doesn't show visual indication (can't use `--default-focus-color` for `border-color` here since it matches)
- [x] lack of focus indicator on Toggle.Switch (color is there but `display` property needs updating)

**Comment by zdavis** [comment]
Thanks for catching this, Dana. I'm planning on doing a 4.1.1 release early next week, and we can get this in it.
